### DISCUSSION TITLE
Let every Manager create a Blog Post

### DISCUSSION THREAD

**Discussion by dopiedopeman** [nmelhado/league-page#180] [General]
Is there a way that every Manager can create a blog post? In our league the one with the fewest points has to create it. So I need to share the contenftul login to everybody or is there an easier way?

**Comment by nmelhado** [comment]
You can add multiple people as contributors to the Contentful account. We have co-commissioners and both of us have access through our own account.

**Reply by dopiedopeman** [reply]
Thx. I got another question. Maybe you can help me with this too. I'm trying to change the input for the Sleeper User Name in the comment section to a dropdown list. Is this possible? I can't find the right file and option for it.

**Reply by nmelhado** [reply]
The input is located in src/lib/BlogPosts/CreateComment.svelte. It should be on line 53. It shouldn't take too much work to convert it to a drop-down and populate it using the user query. The original thought was that it would prevent non-league members from commenting, but that might have been unnecessary.
